content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+## Apply terrain materials
+
+There are two types of materials that are needed for terrain, the *macro material* and *detail materials*.
+
+The macro material is a basic material that applies to the entire terrain. It supports a simple color texture and a normal texture. The macro material is displayed on terrain that is far from the camera.
+
+Detail materials are standard materials that you can create with [Material Editor](/docs/atom-guide/look-dev/materials/material-editor/). They are assigned to the terrain through surface tag names that are generated by gradients, similar to how this tutorial used an image gradient to create elevation. This allows you to assign many materials across a large terrain surface and blend between them. Detail materials are displayed on terrain surfaces that are close to the camera.
+
+{{< tip >}}
+In the **Terrain World Renderer** level component, there is a property group named **Detail material configuration** that you can use to configure how far from the camera to switch from detail material to macro material, and the distance over which to blend between the materials.
+{{< /tip >}}
+
+The following example image demonstrates various material assignments and how they interact on a terrain surface:
+
+{{< image-width src="/images/learning-guide/tutorials/environments/terrain-materials-example.png" width="800" alt="An example of macro and detail materials applied to a terrain." >}}
+
+In this example, there are four materials:
+
+* Magenta - This is th macro material. It's displayed on terrain that is 512 units from the camera. The blend distance between this macro material and the detail materials is 54 units.
+
+* Green - This is the default detail material. Detail materials are layered, and the default material is the base layer. The default material is displayed on areas that aren't aren't assigned a detail material through a surface tag name, but are close to the camera.
+
+* Red - This is a detail material. It's applied to the terrain through a surface tag name. The surface tag name is generated by a tiled image gradient that contains a white triangle on a black background.
+
+* Blue - This is a detail material. It's applied to the terrain through a surface tag name. The surface tag name is generated by a tiled image gradient that contains a white circle on a black background.
+
+{{< note >}}
+Notice that the red and blue detail materials blend to purple in areas where the triangles and circles overlap. The is because the image gradients are generating surface tags for both materials in these overlap areas.
+{{< /note >}}
+
+### Apply a macro material
+
+1. With the Terrain Spawner entity selected, add a **Terrain Macro Material** component.
+
+1. In the **Terrain Macro Material** component, supply a color texture and a normal texture.
+
+{{< note >}}
+If no detail materials are applied to the terrain, the macro material is also displayed on terrain surfaces that are close to the camera.
```

Comment:
The only time the macro material _won't_ be blended close to the camera is if the detail materials aren't authored to blend. (i.e. the Texture Blend Mode on the material is set to "Lerp" instead of "LinearLight" or "Overlay")

Reply:
Removed this note.